.env.local

```
@@ -1,20 +1,33 @@
 API_KEY=UHp3aKb40fwpoKZluZByWQ
+API_URL=http://localhost:9000
+JWT_SECRET=lFW5zCIyU/oFymIoOTSvIl/xfZaKWFvAVDRL+9S7OeA=
 MONGODB_URL_TESTS=mongodb://root:MheCk6sSKB1m4xKNw5I@localhost/cb_faynosync_db_tests?authSource=admin
 PERFORMANCE_MODE=true
 REDIS_HOST=localhost
 REDIS_PORT=6379
 REDIS_PASSWORD=
 REDIS_DB=0
-ENABLE_PRIVATE_APP_DOWNLOADING=false
\ No newline at end of file
+ENABLE_PRIVATE_APP_DOWNLOADING=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ENABLE_PRIVATE_APP_DOWNLOADING key should go before the JWT_SECRET key

(UnorderedKey)